torch/_dynamo/config.py

```diff
@@ -381,6 +381,21 @@ def _get_optimize_ddp_mode():
 # torch._dynamo.utilsCompileTimeInstructionCounter.
 record_compile_time_instruction_count = False
 
+# Number of eager warmup runs to run before compiling.
+# NOTE: To have warmup runs for Compiled Autograd, you must reuse the same Compiled Autograd context manager
+# (where we will do the stateful tracking of the warmup count).
+# Example:
+# ```
+# torch._dynamo.config.warmup_runs = 1
+# ctx = torch._dynamo.compiled_autograd.enable(torch.compile)
```

Comment:
if it works for both APIs, let's suggest the torch.compile one

torch._dynamo.config.compiled_autograd = True
torch.compile(loss.backward)()